ocb3/src/lib.rs

```diff
@@ -0,0 +1,602 @@
+#![no_std]
+#![cfg_attr(docsrs, feature(doc_cfg))]
+#![doc = include_str!("../README.md")]
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg",
+    html_favicon_url = "https://raw.githubusercontent.com/RustCrypto/meta/master/logo.svg"
+)]
+#![deny(unsafe_code)]
+#![warn(missing_docs, rust_2018_idioms)]
+#![allow(non_snake_case)]
```

Comment:
I would prefer for Rust code to follow the naming convention, even if it means using slightly different variable names from the spec.